Dockerfile

```diff
@@ -8,18 +8,6 @@ LABEL "com.github.actions.description"="Deploy your DNS configuration to multipl
 LABEL "com.github.actions.icon"="cloud"
 LABEL "com.github.actions.color"="yellow"
 
-ENV DNSCONTROL_VERSION="3.31.4"
-ENV DNSCONTROL_CHECKSUM="054d236531df2674c9286279596f88f02c1cf7b1448dc5f643f1a1dbe705fe8d"
-
-RUN apk -U --no-cache upgrade && \
-    apk add --no-cache bash ca-certificates curl libc6-compat
-
-RUN curl -sL "https://github.com/StackExchange/dnscontrol/releases/download/v$DNSCONTROL_VERSION/dnscontrol-Linux" \
-  -o dnscontrol && \
-  echo "$DNSCONTROL_CHECKSUM  dnscontrol" | sha256sum -c - && \
-  chmod +x dnscontrol && \
-  mv dnscontrol /usr/local/bin/dnscontrol
-
 RUN ["dnscontrol", "version"]
```

Comment:
The stackexchange/dnscontrol doesn't come with bash which is required for `entrypoint.sh`. In my testing changing this run line to include an `apk add` for bash fixes this.

```dockerfile
RUN dnscontrol version && \
    apk add --no-cache bash
```